src/index.js

```diff
@@ -4,22 +4,26 @@ import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
 import {BrowserRouter} from "react-router-dom"
-import { FilterProvider,CartProvider,WishListProvider } from "./context";
+import { FilterProvider,CartProvider,WishListProvider, AuthProvider, ToastProvider } from "./context";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
-    <FilterProvider>
-      <CartProvider>
-        <WishListProvider>
-          <BrowserRouter>
-            <App />
-          </BrowserRouter>
-        </WishListProvider>
-      </CartProvider>
-    </FilterProvider>
+    <ToastProvider>
```

Comment:
You can have a separate file for these contexts and then add import that single component here .